libapicore/ApiServer.cpp

```diff
@@ -4,6 +4,7 @@
 ApiServer::ApiServer(AbstractServerConnector *conn, serverVersion_t type, Farm &farm, bool &readonly) : AbstractServer(*conn, type), m_farm(farm)
 {
 	this->bindAndAddMethod(Procedure("miner_getstat1", PARAMS_BY_NAME, JSON_OBJECT, NULL), &ApiServer::getMinerStat1);
+	this->bindAndAddMethod(Procedure("miner_getstat2", PARAMS_BY_NAME, JSON_OBJECT, NULL), &ApiServer::getMinerStat2);	
```

Comment:
Please do not use ```miner_getstat2```  this is already used by newer claymore versions for extended results.
Please use something else. Also since we are not restricted by claymore compatibility, you could even return a nicer output. for example json objects for each gpu, with fields like "power" or "temp".

Reply:
I'll create getMinerStatReadable instead of that one then! That one will return a more human-readable format. Do you know of any common format that some miners use or something?